### PR TITLE
adds pagetree fixes for other themes

### PR DESCRIPTION
**Pull request by lory87**
|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-02 at 16 15 38|screen shot 2015-11-03 at 09 46 26|

### CODE REVIEW

File: cms/static/cms/sass/cms.pagetree.scss [hunk 1]
```diff
@@ -423,6 +425,10 @@ $row-height: 41px;
         display: none;
     }
 }
+#changelist-search {
```
Comment (darioalbanesi) [inline cms/static/cms/sass/cms.pagetree.scss]: why ID?

**Comment by ruslanmalogulko** [comment]
:+1:

File: cms/static/cms/sass/cms.pagetree.scss (hunk 1 above)
Comment (lory87) [inline cms/static/cms/sass/cms.pagetree.scss]: because there is no class and I have to work with the given html.

**Comment by darioalbanesi** [comment]
lgtm

**Comment by vxsx** [comment]
:+1: (still wait for travis)

**Comment by FinalAngel** [comment]
@vxsx didn't we wanted to use border box everwhere? :D

**Comment by vxsx** [comment]
@FinalAngel we did, but not in the admin (and it's a bit late :D)

**Pull request merged**